docs/index.rst

```diff
@@ -8,17 +8,21 @@ Welcome to broadbean's documentation!
 The broadbean package is a tool for creating and manipulating pulse
 sequences.
 
-=====================================
 
-.. toctree::
-   :maxdepth: 2
-   :caption: Contents:
-
-.. automodule:: broadbean.broadbean
-    :members:
+Documentation
+-------------
 
-.. automodule:: broadbean.ripasso
-    :members:
+.. toctree::
+    :maxdepth: 2
+
+    api/generated/broadbean
+    examples/Pulse_Building_Tutorial.ipynb
+    examples/Making_output_for_Tektronix_AWG70000A.ipynb
+    examples/Example_Write_Read_JSON.ipynb
+    examples/Filter_compensation.ipynb
+    examples/Subsequences.ipynb
```

Comment:
Rather than including all the examples directly on the front page I think its better to create a subpage for them like in qcodes https://github.com/QCoDeS/Qcodes/blob/master/docs/examples/index.rst

Reply:
sounds good I'll make that change